spring-integration-core/src/main/java/org/springframework/integration/config/AbstractSimpleMessageHandlerFactoryBean.java

```diff
@@ -122,6 +124,14 @@ public void setAdviceChain(List<Advice> adviceChain) {
 		this.adviceChain = adviceChain;
 	}
 
+	/*
+	 * Supported at the super class level but only exposed on the service activator
+	 * namespace. It's not clear that other endpoints would benefit from async support.
+	 */
```

Comment:
OK. That makes sense, too.
But I'm sure it can become a JavaDoc, since this method is `public`. 
Or even must!

With the growing community there is a big chance that someone will complain that we don't have a JavaDoc on this method :wink: